src/Dofs/MixedDofHandler.jl

```diff
@@ -9,9 +9,9 @@ end
 
 Construct a `FieldHandler` based on an array of `Field`s and assigns it a set of cells.
 """
-function FieldHandler(fields::Vector{Field}, cellset)
+function FieldHandler(fields::Vector{Field}, cellset, ipgeo::Vector{ip}=getproperty.(fields,:interpolation)) where ip <: Interpolation
```

Comment:
Geometry interpolation is called `geom_interpol` in most of the files, maybe be consistent with that?